### PR TITLE
Ansys em 2020r1

### PR DESCRIPTION
**Pull request by orbsmiv**
N.B. requires https://github.com/bear-rsg/easybuild-easyblocks/pull/16

`ANSYSEM-2020R1-GCCcore-8.3.0.eb`
* [x] Assigned to reviewer
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] Ubuntu16 VM

### CODE REVIEW

File: easybuild/easyconfigs/a/ANSYSEM/ANSYSEM-2020R1-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+]
+
+sources = [SOURCE_ZIP]
+checksums = ['1a58882516ad59d2ad7c37c0d8e161f85eac55355221b835f048bc573c43a659']
+
+# Licensing information can be provided using the following configuration variables, else the following
+# environment variables will be interrogated: EB_ANSYS_EM_LICENSE_SERVER & EB_ANSYS_EM_LICENSE_SERVER_PORT
+
+license_server = "server.local"
+license_server_port = "1234"
```
Comment (branfosj) [inline easybuild/easyconfigs/a/ANSYSEM/ANSYSEM-2020R1-GCCcore-8.3.0.eb:27]:
```suggestion
# license_server = "server.local"
# license_server_port = "1234"
```

**Comment by branfosj** [comment]
You'll want something like:
```python
        # ANSYSEM and deps, fine so long as libpng 1.2.58 is in the versionsuffix
        if dep in [] and len(dep_vars) > 1:
            libpng1_2_58_vars = [v for v in dep_vars.keys() if v.endswith('versionsuffix: -libpng-1.2.58')]
            if len(libpng1_2_58_vars) == 1:
                dep_vars = dict((k, v) for (k, v) in dep_vars.items() if k != libpng1_2_58_vars[0])
```

File: easybuild/easyconfigs/a/ANSYSEM/ANSYSEM-2020R1-GCCcore-8.3.0.eb (hunk 1 above)
Comment (orbsmiv) [reply easybuild/easyconfigs/a/ANSYSEM/ANSYSEM-2020R1-GCCcore-8.3.0.eb:27]: This is intentional (see beda521) and is to ensure that the installation doesn't try to use our currently defined licence env vars (`EB_ANSYS_EM_LICENSE_SERVER` & `EB_ANSYS_EM_LICENSE_SERVER_PORT`), which I will ultimately remove from the dev/live build modules.
Instead, we want ANSYS / ANSYSEM to use its globally defined licence env vars at runtime:
```
ANSYSLI_SERVERS=2325@lic-ansys.bham.ac.uk
ANSYSLMD_LICENSE_FILE=1055@lic-ansys.bham.ac.uk
ANSOFTD_LICENSE_FILE=1055@lic-ansys.bham.ac.uk
```

**Comment by orbsmiv** [comment]
Built on all requested platforms

**Pull request merged**